Integrations/Kafka_V2/Kafka_V2.py

```diff
@@ -0,0 +1,392 @@
+            else:
+                return_error('Supplied offset is not a number')
+            if check_latest_offset(topic, partition_number=partition) <= offset:
+                return_error('Offset is out of bounds')
+        else:
+            return_error('Offset is not a number, earliest or latest')
+    return offset, partition
+
+
+def create_incident(message, topic):
+    """
+    Creates incident
+    :param message: Kafka message to create incident from
+    :type message: :class:`pykafka.common.Message`
+    :param topic: Message's topic
+    :type topic: str
+    :return incident:
+    """
+    raw = {
+        'Topic': topic,
+        'Partition': message.partition_id,
+        'Offset': message.offset,
+        'Message': message.value
+    }
+    return {
+        'name': 'Kafka {} partition:{} offset:{}'.format(topic, message.partition_id, message.offset),
+        'occurred': timestamp_to_datestring(int(time.time())),
+        'details': message.value,
+        'rawJSON': json.dumps(raw)
+    }
+
+
+def check_latest_offset(topic, partition_number=None):
+    """
+    :param topic: topic to check the latest offset
+    :type topic: :class:`pykafka.topic.Topic`
+    :param partition_number: partition to take latest offset from
+    :type partition_number: int, str
+    :return latest_offset: last message offset
+    :rtype: int
+    """
+    partitions = topic.latest_available_offsets()
+    latest_offset = 0
+    if partition_number is not None:
+        partition = partitions.get(str(partition_number))
+        if partitions:
+            latest_offset = partition[0][0]
+        else:
+            return_error('Partition does not exist')
+    else:
+        for partition in partitions.values():
+            if latest_offset < partition[0][0]:
+                latest_offset = partition[0][0]
+    return latest_offset - 1
+
+
+def create_certificate():
+    """
+    Creating certificate
+    :return certificate:
+    :return type: :class: `pykafka.connection.SslConfig`
+    """
+    ca_path = 'ca.cert'  # type: ignore
+    client_path = 'client.cert'
+    client_key_path = 'client_key.key'
+    if CA_CERT:
+        with open(ca_path, 'wb') as file:
+            file.write(CA_CERT)
+            ca_path = os.path.abspath(ca_path)
+        if CLIENT_CERT:
+            with open(client_path, 'wb') as file:
+                file.write(CLIENT_CERT)
+                client_path = os.path.abspath(client_path)
+        else:
+            client_path = None  # type: ignore
+        if CLIENT_CERT_KEY:
+            with open(client_key_path, 'wb') as file:
+                file.write(CLIENT_CERT_KEY)
+        else:
+            client_key_path = None  # type: ignore
+    else:
+        ca_path = None  # type: ignore
+
+    return SslConfig(
+        cafile=ca_path,
+        certfile=client_path,
+        keyfile=client_key_path,
+        password=PASSWORD
+    )
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def test_module():
```

Comment:
revert the change here + in the next PR please give more informative error, the user should understand what happend